google/cloud/logging_v2/handlers/structured_log.py

```diff
@@ -16,10 +16,13 @@
 """
 import collections
 import json
+import logging
```

Comment:
When using StructuredLogHandler, the log passed through Python's logging std lib before reaching our library. Would that be worth capturing as an instrumentation source? Do we want to be able to differentiate between different handlers?

Reply:
I think the spec would allow for that, but that might be overkill for the first version of instrumentation source.  I think just knowing overall library usage and version numbers is the primary goal.  But I'll defer to @losalex on this.

Reply:
FWIW I noticed nodejs seems to be [capturing both node and the bunyan integration library info](https://github.com/googleapis/nodejs-logging-bunyan/blob/101822bc4ec43d579287bdf02846f5b62803f76a/system-test/logging-bunyan.ts#L86)

But yeah, we can probably decide if we want to increase the resolution of the instrumentation later. It may be worth thinking about the naming choices for the different layers of instrumentation should be named ahead of time though, because that could be hard to change later. But I think using `python` should probably work for the base entry

Reply:
We should capture all libraries we own.

Reply:
I think for now we probably don't need to capture std lib logging then.  It might be useful later, but for now the focus is on the layers we control.

Reply:
I do think the integration layer for the stdlib is kind of a second library on its own, even though it's in the same repo. And I think it would be very useful to see how many people are using the integration vs using the API directly

But yeah agreed, that can easily be added later

Reply:
I would say we should look into it if you think it is important - I probably should mention before that libraries we own is a must, but others are definitely subjects to consider. Lets sync on this